live-build/misc/upgrade-scripts/common.sh

```diff
@@ -15,7 +15,10 @@
 # limitations under the License.
 #
 
+# shellcheck disable=SC2034
 UPDATE_DIR=${UPDATE_DIR:-"/var/dlpx-update"}
+UPDATE_VERIFY_LOG_DIR=${UPDATE_DIR}/verify-logs
```

Comment:
I'm not necessarily against putting these logs in `/var/dlpx-update`, but we need to just be aware that this directory is backed by the `rpool/update` dataset, which has a quota on it. Thus, if we fill this directory with logs (I don't know what the disk usage is of these logs), it may mean we can't hold as many upgrade images as we previously could.

I don't know if the upgrade images + logs will cause ENOSPC issues, but I want to make sure we think that through. If we can hold both in this dataset, I'm OK with this approach.

Reply:
Good point. I collected support bundle (just the `log` and `dropbox` type) from longevity setup and the compressed size is 508M. The logs are currently rotating so I presume it's near the upper bound of a bundle size for those two types. I think saving one is OK, although we have to rethink different place to store this if we decide to store more than one.

Do you have any good location in mind where to save these logs? How about `/var/delphix/server/log`?